src/cowrie/commands/yum.py

```diff
@@ -1,9 +1,3 @@
-# Copyright (c) 2009 Upi Tamminen <desaster@gmail.com>
```

Comment:
Possible that this is a wrong commit? ^^

Reply:
Oh yes. Even though I really don't like the code - bloating with this useless copyrights.

Reply:
I agree, that's also something on my bucket list.